lib/js/jslib.js

```diff
@@ -1761,31 +1524,26 @@ function _swapNodes(x, y) {
   if(yNextSibling != x)
     yParent.insertBefore(x, yNextSibling);
 
-  return {}
+  return CONSTANTS.UNIT;
 }
 var swapNodes = LINKS.kify(_swapNodes);
 
-//replaceChildren : xml -> domRef -> ()
-function _replaceChildren(xml, parent) {
+/**
+ * domReplaceChildren : xml -> domRef -> ()
+ * @param {Xml} xml
+ * @param {DomNode} parent
+ * @returns {()}
+ */
+function _domReplaceChildren(xml, parent) {
   newNodes = LINKS.XmlToDomNodes(xml);
```

Comment:
Not necessarily `const`, `var` or `let` would do as well, but yeah, it needs to be declared.
That is a bit of a problem with JS. Unless the line is actually executed you don't know whether it is broken. So I might have overlooked some other instances of undeclared variables that were not executed yet. =/